pkg/blocks/expectstep.go

```diff
@@ -52,7 +51,6 @@ type ExpectStep struct {
 	Executor       string                  `yaml:"executor,omitempty"`
 	Expect         *ExpectSpec             `yaml:"expect,omitempty"`
 	Environment    map[string]string       `yaml:"env,omitempty"`
-	CleanupStep    string                  `yaml:"cleanup,omitempty"`
```

Comment:
@l50 Why did we introduce the text version of clean up to this step in the first place?

Reply:
Hey! When I was coding this up, I employed the filestep and editsteps as the guiding force for I should put a new step together.

It looks like some things have changed since, so there's no explicit reason - I was merely copying and modifying what we already had